pkg/destroy/ibmcloud/ibmcloud.go

```diff
@@ -173,47 +174,55 @@ func (o *ClusterUninstaller) executeStageFunction(f struct {
 
 func (o *ClusterUninstaller) loadSDKServices() error {
 	apiKey := os.Getenv("IC_API_KEY")
-	authenticator := &core.IamAuthenticator{
-		ApiKey: apiKey,
-	}
-
-	err := authenticator.Validate()
```

Comment:
When a new service is created, `Validate` is called as part of the `BaseService` instantiation.